master/docs/manual/cfg-workers-openstack.rst

```diff
@@ -152,6 +152,28 @@ When the instance is destroyed, the volume will be terminated as well.
     ]
 
 
+The ``nova_args`` can be used to specify additional arguments for the novaclient. For example network mappings,
+which is required if your openstack tenancy has more than one network, and default cannot
+be determined. Please refer to your openstack manual whether it wants net-id or net-name.
+
+Other useful parameters are availability_zone, security_groups and config_drive. Refer to
+`Python bindings to the OpenStack Nova API <http://docs.openstack.org/developer/python-novaclient/>`
```

Comment:
I think rst requires a _ in the end: 

   ...openstack.org/developer/python-novaclient/>`_